### DISCUSSION TITLE
关于chaiNNer不出现UI界面的问题

### DISCUSSION THREAD

**Discussion by ZTKSUB** [chaiNNer-org/chaiNNer#3022] [Q&A]
我装好了chaiNNer Windows,但是他的UI界面不弹出,但是Windows任务管理器看,会看到chaiNNer的进程,请问这种情况该如何解决
![QQ截图20240914134947](https://github.com/user-attachments/assets/e34354a8-8835-422d-94d2-6dda07479f98)

**Comment by joeyballentine** [comment]
Maybe something to do with chinese system language or using VPN. impossible to know without seeing logs

**Reply by ZTKSUB** [reply]
[Squirrel-Shortcut.log](https://github.com/user-attachments/files/17008472/Squirrel-Shortcut.log)
是这个吗
